src/interfaces/ExtendedRequest.ts

```diff
@@ -0,0 +1,9 @@
+import { UrlWithParsedQuery } from 'url';
```

Comment:
implement yours

Reply:
this is Node native module, no problems with using it